src/lara-app/utils/generate-dataset.test.tsx

```diff
@@ -47,12 +52,22 @@ describe("generateDataset helper", () => {
       type: "dataset",
       version: 1,
       properties: ["Location", "Temperature (\u00B0C)", "Humidity (%)", "Illuminance (lux)"],
-      xAxisProp: "Location", // 1st prop
+      xAxisProp: "Location",
       rows: [
         ["corner1", 1, 10, 100],
         ["corner2", 2, 11, 110],
         ["corner3", 3, 12, 120]
       ]
     });
   });
+
+  // This test should ensure that when new epxeriment is added, developer won't miss xAxisPropertyForExperiment hash.
```

Comment:
`epxeriment` => `experiment`

Reply:
Thanks, fixed in https://github.com/concord-consortium/vortex/commit/ba89ef76c856980d798cf2d01d912c280f250735